src/plugins/kibana_utils/public/state_management/url/kbn_url_storage.test.ts

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
Initially it was without `Kbn` prefix, but I reconsidered and added `Kbn` just to emphasise this is *type of storage* is something which is coming from legacy Kibana and Kibana App. 

I thought that later we could have differently named url storage, which would allow to preserve state in nicer and more simple user friendly format (e.g by serialising using regular query params like &query="test"&time="test&filter="filter1,filter2"), where `Kbn` is something took from old Kibana, with rison, storing in hash part of url and other details.